scripts/run_tests.sh

```
@@ -0,0 +1,8 @@
+#!/bin/bash
+CURRENT_DIR="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+PARENT_DIR="$( builtin cd ${CURRENT_DIR}/.. >/dev/null 2>&1 ; pwd -P )"
+cd ${PARENT_DIR}
+
+go test -v -timeout=5h ./... 2>&1 | tee tests_output.log
```

```suggestion
go test -v -failfast -timeout=5h ./... 2>&1 | tee tests_output.log
```

---

Do we want to fail fast? What if we want to ignore a check and see what else fails (e.g. the IncCounter check that fails now?)

---

that's how I usually operate the cluster tests take so long and are so verbose that I just stop at the first failure, then re-run. most tests results are cached anyway.

Feel free to ignore